content/applications/essentials/search.rst

```diff
@@ -183,6 +183,68 @@ It is possible to *customize groups* by using a field present on the model. To d
 
 .. _search/favorites:
 
+Comparison
+==========
+
+Certain reporting dashboards include a :guilabel:`Comparison` section in the drop-down menus of
+their :guilabel:`Search...` bars. This includes the :doc:`Overall Equipment Effectiveness
+<../inventory_and_mrp/manufacturing/workflows/oee>` report for the *Manufacturing* app, and the
+:doc:`Purchase <../inventory_and_mrp/purchase/advanced/analyze>` report for the *Purchase* app,
+among others.
+
+The options in the :icon:`fa-adjust` :guilabel:`Comparison` section are used to compare data from
+two different time periods. There are two comparison options to choose from: :guilabel:`(Time
+Filter): Previous Period` and :guilabel:`(Time Filter): Previous Year`.
+
+.. important::
+   For some reports, the :guilabel:`Comparison` section only appears in the :guilabel:`Search...`
+   bar drop-down menu if one or more time periods is selected in the :guilabel:`Filters` column.
+   This is because, if no time period is specified, there is nothing to make a comparison to.
+
+   Additionally, some reports only allow use of the :guilabel:`Comparison` feature when the
+   :icon:`fa-pie-chart` :guilabel:`(pie chart)` graph type or the :icon:`oi-view-pivot`
+   :guilabel:`(pivot)` view is selected. A :guilabel:`Comparison` option can be selected even if
+   another view is enabled, but doing so does not change the way data is displayed on the report.
+
+.. image:: search/comparison-section.png
+   :align: center
+   :alt: The Search... bar for the production analysis report.
+
+To view data using one of the two comparisons, begin by selecting a time period in the
+:guilabel:`Filters` column of the :guilabel:`Search...` bar drop-down menu. Then, select either
+:guilabel:`(Time Filter): Previous Period` or :guilabel:`(Time Filter): Previous Year` in the
+:guilabel:`Comparison` section.
+
+With one of the :guilabel:`Comparison` options enabled, the report compares the data for the
+selected period, with the data for the same unit of time (month, quarter, year), one period or year
+prior. The way the data is displayed depends on the selected view:
+
+- The bar chart shows two bars side-by-side for each time unit of the selected time period. The left
+  bar represents the selected time period, while the right bar represents the previous time period.
+- The line chart is displayed with two lines, one representing the selected time period, and the
+  other representing the previous time period.
+- The pie chart appears as a large circle with a smaller circle inside of it. The larger circle
+  represents the selected time period, while the smaller circle represents the previous time period.
+- The pivot table is displayed with each column split into two smaller columns. The right column
+  represents the selected time period, while the left column represents the previous time period.
```

Comment:
It might be nice to add icons & GUI(parentheticals) to each of these view types being discussed. Could add a nice visual element to this portion of the doc. Just something to consider during your revisions.